app/test_setup.go

```diff
@@ -36,8 +45,15 @@ func InitElysTestApp(initChain bool) *ElysApp {
 		MakeEncodingConfig(),
 		simapp.EmptyAppOptions{},
 	)
+
+	return app
+}
+
+// Initializes a new ElysApp without IBC functionality
+func InitElysTestApp(initChain bool) *ElysApp {
```

Comment:
InitElysTestApp and InitiateNewElysApp seems to be a similar meaning - could you clarify which one is mainly used on testing? (If one is not used e.g. InitiateNewElysApp  - it could be updated to initiateNewElysApp so that it's not for use from other package)

Reply:
@jelysn 
Yeah you are right. They are almost the same but the second one is just returning the genesis account. The genesis account is the genesis delegator of the genesis validator. I just needed it to test the function of calculating delegation amount. So I added a new function of initiating testapp that returns genesis account.